strawberryfields/apps/graph/utils.py

```diff
@@ -156,73 +156,75 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
 
 
 def is_clique(graph: nx.Graph) -> bool:
-    """Determines if the input graph is a clique
+    """Determines if the input graph is a clique. A clique of n nodes has n*(n-1)/2 edges
 
     Args:
-        graph (nx.Graph): the input graph
+        graph (nx.Graph): The input graph
 
     Returns:
-        bool: returns ``True`` if input graph is a clique and ``False`` otherwise
+        bool: Returns ``True`` if input graph is a clique and ``False`` otherwise
```

Comment:
Actually, we can just drop `Returns` here

Reply:
Done!